packages/calcite-components/src/components/action/action.tsx

```diff
@@ -299,7 +299,10 @@ export class Action
       buttonId,
       messages,
     } = this;
-    const ariaLabel = `${label || text}${indicator ? ` (${messages.indicator})` : ""}`;
+    const labelFallback = label || text;
+    const ariaLabel = labelFallback
+      ? `${labelFallback}${indicator ? ` (${messages.indicator})` : ""}`
```

Comment:
This should use a localized template. Since this is already in production, can you create an issue? We can link to action’s doc site page for the repro case since this is a screen reader label.

Reply:
created https://github.com/Esri/calcite-design-system/issues/9071